ryanoasis/nerd-fonts
# master branch changes for #160

#### Description

I've added a CONTRIBUTORS.md file that includes a table of all contributors in all-contributors format. It also has a badge indicating the number of contributors. This is only the first part of the changes needed for #160 

There is also a .all-contributorsrc file that is used with the [all-contributors-cli](https://github.com/jfmengels/all-contributors-cli) for easy addition of new contributors.

I think this is a good starting point. I tried to match up everyone's actual contributions to an emoji or multiple emojis. Please change them if they aren't correct! Any fonts added I just lumped in with the "code" emoji.

There will be a part 2 PR for #160 to the gh-pages branch but I have to change the format because jekyll doesn't like the markdown used for the table generated by the [all-contributors-cli](https://github.com/jfmengels/all-contributors-cli). I may attempt automating the conversion with a script.

#### Requirements / Checklist

  - If any of the scripts were modified they have been tested and execute without error, e.g.:
    - `./font-patcher Inconsolata.otf --fontawesome --octicons --pomicons`
    - `./gotta-patch-em-all-font-patcher\!.sh Hermit`

#### What does this Pull Request (PR) do?
- add CONTRIBUTORS.md
#### How should this be manually tested?
- Click [here](https://github.com/lucasreed/nerd-fonts/blob/master/CONTRIBUTORS.md) to see what it looks like in my fork.
#### Any background context you can provide?

#### What are the relevant tickets (if any)?
- #160 
#### Screenshots (if appropriate or helpful)
